validator/src/main/java/com/amazon/aoc/validators/CWLogValidator.java

```diff
@@ -145,7 +147,13 @@ private Map<String, Object> getActualLog(
       dependencyFilter += String.format(" && ($.RemoteResourceType = %%%s%%) && ($.RemoteResourceIdentifier = %%%s%%)", remoteResourceType, remoteResourceIdentifier);
     }
 
-    String filterPattern = String.format("{ ($.Service = %s) && ($.Operation = \"%s\") %s }", context.getServiceName(), operation, dependencyFilter);
+    if (operation != null) {
+      dependencyFilter += String.format(" && ($.Operation = \"%s\")", operation);
+    } else {
+      dependencyFilter += "&& ($.Operation NOT EXISTS)";
+    }
```

Comment:
nit: A comment explaining why we wouldn't need operation during validation would be helpful

Reply:
Added